src/sce-elf-defs.h

```diff
@@ -41,19 +41,29 @@ typedef struct SCE_TYPE(sce_module_info) {
 	SCE_PTR(const void *) extab_end;	/* Offset to end of ARM EXTAB (optional */
 
 	// i decided to include process param into module_info (xyz)
-	uint32_t process_param_size;
-	uint32_t process_param_magic;
-	uint32_t process_param_ver;
-	uint32_t process_param_fw_ver;
+	uint32_t              process_param_size;                         // 0x34
+	uint32_t              process_param_magic;                        // PSP2
+	uint32_t              process_param_ver;
+	uint32_t              process_param_fw_ver;
 	SCE_PTR(const char *) process_param_main_thread_name;
-	int32_t process_param_main_thread_priority;
-	uint32_t process_param_main_thread_stacksize;
-	uint32_t process_param_main_thread_attribute;
+	SCE_PTR(int32_t    *) process_param_main_thread_priority;
+	SCE_PTR(uint32_t   *) process_param_main_thread_stacksize;
+	uint32_t              process_param_main_thread_attribute;
```

Comment:
it doesn't need spaces.
could you just add `SCE_PTR(int32_t *)`, `SCE_PTR(uint32_t *)` and comments?